### DISCUSSION TITLE
Are the artwork in phaser example repos free to use?

### DISCUSSION THREAD

**Discussion by debola31** [photonstorm/phaser#5961] [Q&A] [answered]
I'm developing a web based video game to be used internally at my company. It's not going to be sold to anyone and will only be used internally. I want to make sure all the artwork I'm using isn't copyrighted. I use a lot of the sprites and assets that are found on the phaser examples website https://phaser.io/examples. Does anyone know if these are free to use or copyrighted?. My management is having me doublecheck this.

**Comment by photonstorm** [comment, marked as answer]
It depends on the asset specifically, but while some of them are free, lots come from old arcade and console games, so they likely still fall under the copyrights of their original owners.
